plugins/accounts/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>hawkular-accounts</title>
+    <base href='/'>
+    <meta charset="UTF8">
+
+    <link rel="stylesheet" href="libs/bootstrap/dist/css/bootstrap.css" />
+    <link rel="stylesheet" href="libs/patternfly/dist/css/patternfly.css" />
+
+    <!-- bower:css -->
+    <!-- endbower -->
+
+    <!-- ES6/ES6 shim -->
+    <!--[if IE 8]>
+    <script src="libs/es5-dom-shim/__COMIPLE/a.ie8.js"></script>
+    <![endif]-->
+    <script src="libs/es5-dom-shim/__COMPILE/a.js"></script>
+
+    <!-- bower:js -->
+    <!-- endbower -->
+
+    <script src="libs/bootstrap/dist/js/bootstrap.js"></script>
+    <script src="libs/patternfly/dist/js/patternfly.js"></script>
+
+</head>
+
+<body>
+<nav class="navbar navbar-default navbar-pf" role="navigation">
+    <ul class="nav navbar-nav navbar-primary" hawtio-main-nav></ul>
+    <ul class="nav navbar-nav" hawtio-sub-tabs></ul>
+</nav>
+
+<div id="main" class="container-fluid ng-cloak" ng-controller="HawtioNav.ViewController">
+    <div ng-include src="viewPartial"></div>
+</div>
+
+<!-- add any scripts under dist/ here -->
+<script src="dist/hawkular-ui-components-accounts.js"></script>
+
+<script type="text/javascript">
```

Comment:
Note that the Keycloak integration on this file is not "final" and will be replaced once Keycloak is integrated with hawt.io.